### PR TITLE
JW-1122 controller cleanup

### PR DESCRIPTION
**Pull request by mclapinski**

### CODE REVIEW

File: library/cpp/transport/TFDTransport.cpp [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
 #ifdef _WIN32
 #include <io.h>
-#include <winutils.h>
+typedef ptrdiff_t ssize_t;
```
Comment (sodar) [inline library/cpp/transport/TFDTransport.cpp:31]: Couldn't we use `ptrdiff_t` from `cstddef` header file?

**Review by sodar — CHANGES_REQUESTED** [review]

File: library/cpp/transport/TFDTransport.cpp (hunk 1 above)
Comment (mclapinski) [reply library/cpp/transport/TFDTransport.cpp:31]: Here you go: http://en.cppreference.com/w/cpp/language/typedef

**Review by sodar — APPROVED** [review]

**Comment by mclapinski** [comment]
@jacek-izykowski please review, Dariusz's comment was invalid

**Comment by mclapinski** [comment]
Retest this please

**Review by jacek-izykowski — APPROVED** [review]

**Comment by mclapinski** [comment]
Retest this please

**Pull request merged**